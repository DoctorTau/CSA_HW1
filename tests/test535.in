203
-1453 9374 8257 -5662 7804 1778 9955 1886 6766 -7049 6209 -2263 7538 1402 -6888 3201 -5123 -6720 6501 2453 242 -6878 1573 4149 4844 7549 -4078 -9921 3905 -7316 -9400 -1331 8647 -7995 -5407 4694 403 -439 4958 7650 3235 6857 -3920 -8191 -6415 -7292 -4797 3155 -5490 -3764 7908 -717 7064 -9118 15 -9493 5368 -2147 9277 6389 200 -2500 4397 6052 396 6571 64 549 2390 2068 6168 -9919 -1125 -2859 -6144 -6109 5950 -8671 6064 -9394 3393 -2169 531 7462 8203 2159 8287 664 -5098 -1227 -3834 4630 5084 5925 -1032 4436 -7445 4641 -8156 7154 8951 2918 5853 -3459 -4553 -8149 -3181 7624 -5141 8526 -5014 -2169 719 -1536 560 -5903 9015 8373 8606 5325 -8154 -5260 -6977 2400 415 2733 -4867 -145 -3526 8853 -5480 9847 6341 -2256 -8769 3996 6467 4198 812 -1781 -4358 2792 -6569 2849 -7166 3777 5286 -3652 -9429 -4810 -929 9533 8854 5880 -9677 2174 -9798 330 5661 4520 9102 3591 733 816 -2520 4016 7908 -642 3940 6887 -4007 -4600 -9123 9939 -485 679 -7423 6401 -7162 -5641 -3509 4831 -9544 3025 7503 7127 6471 2783 6201 5217 -6552 -5545 -7473 -2171 4550 2146 3550 -3940 1856 43 -94 -4945 1687 