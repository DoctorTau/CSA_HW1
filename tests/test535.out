-5662 -6888 -6720 -6878 -4078 -7316 -9400 -3920 -7292 -5490 -3764 -9118 -2500 -6144 -9394 -5098 -3834 -1032 -8156 -5014 -1536 -8154 -5260 -3526 -5480 -2256 -4358 -7166 -3652 -4810 -9798 -2520 -642 -4600 -7162 -9544 -6552 -3940 -94 